mbuild/formats/lammpsdata.py

```diff
@@ -651,9 +661,22 @@ def _get_bond_types(
             )
         )
     )
+
+    lx = len(atom_types)
+    bond_sets = [x[2] for _, x in unique_bond_types.items()]
+    ordered_bond_sets = [
```

Comment:
```suggestion
    ordered_bond_tuples = [
```
Just to not get confused that these are python `sets`.